internal/exec/stages/disks/luks.go

```diff
@@ -110,7 +110,9 @@ func (s *stage) createLuks(config types.Config) error {
 		var ignitionCreatedKeyFile bool
 		// create keyfile inside of tmpfs, it will be copied to the
 		// sysroot by the files stage
-		os.MkdirAll(distro.LuksInitramfsKeyFilePath(), 0700)
+		if err := os.MkdirAll(distro.LuksInitramfsKeyFilePath(), 0700); err != nil {
+			return err
```

Comment:
The surrounding code wraps errors before returning them, so we should do that here too.